tests/test_mlx5_cuda_umem.py

```diff
@@ -5,11 +5,15 @@
 
 from pyverbs.providers.mlx5.mlx5dv import Mlx5DevxObj, WqeDataSeg, Mlx5UMEM
 from tests.mlx5_base import Mlx5DevxRcResources, Mlx5DevxTrafficBase
-from tests.mlx5_prm_structs import SwMkc, CreateMkeyIn, CreateMkeyOut
```

Comment:
or you can just move this line into create_mkey().

Reply:
Thanks for taking a look.

This seems the most sensible solution.
I have modified the patch.